Source/Core/DolphinQt2/Settings.h

```diff
@@ -51,4 +51,18 @@ class Settings final : public QSettings
   bool GetRenderToMain() const;
   bool GetFullScreen() const;
   QSize GetRenderWindowSize() const;
+
+  // Columns
+  bool& IsBannerVisible() const;
```

Comment:
Then the function needs to be renamed. You can't expect to have an "Is" function to have a side effect that changes the state.
`bannerVisible()` would be acceptable to me.